go/database/mpt/tool/export.go

```diff
@@ -64,23 +68,51 @@ func doExport(context *cli.Context) error {
 
 	start := time.Now()
 	logFromStart(start, "export started")
+
+	ctx, cancel := context.WithCancel(cliCtx.Context)
+	isCanceled := catchInterrupt(ctx, cancel, start)
```

Comment:
you can use the method you already defined to check the context was interrupted (called now `interrupt`) and then you can avoid `isCanceled` property

Reply:
Right. Good idea!